docs/general/profiles.md

```diff
@@ -0,0 +1,110 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Logs
+aliases: [docs/specs/semconv/general/profiles-general]
+--->
+
+# General Profiles Attributes
+
+**Status**: [Experimental][DocumentStatus]
+
+<!-- toc -->
+
+- [Build ID](#build-id)
+- [Spans and Traces](#spans-and-traces)
+- [Frame types](#frame-types)
+- [Reserved attributes](#reserved-attributes)
+
+<!-- tocstop -->
+
+The attributes described in this section are rather generic.
+They may be used in any Profiles record they apply to.
+
+## Build ID
+
+**Description:** Unique identifier for an executable.
+
+<!-- semconv profile.build_id -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`profile.build_id.gnu`](/docs/attributes-registry/profile.md) | string | The GNU build ID as can be found in the `.note.gnu.build-id` ELF section. | `c89b11207f6479603b0d49bf291c092c2b719293` | `Opt-In` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`profile.build_id.go`](/docs/attributes-registry/profile.md) | string | The Go build ID. | `foh3mEXu7BLZjsN9pOwG/kATcXlYVCDEFouRMQed_/WwRFB1hPo9LBkekthSPG/x8hMC8emW2cCjXD0_1aY` | `Opt-In` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+## Spans and Traces
+
+**Description:** Connect Profile information to Spans and Traces.
```

Comment:
It seems profiles will have dedicated properties for span-id and trace-id https://github.com/open-telemetry/opentelemetry-proto/blob/40b3c1b746767cbc13c2e39da3eaf1a23e54ffdd/opentelemetry/proto/profiles/v1experimental/pprofextended.proto#L94

What would be the use-case for `profile.link.*`? what they will be stamped on? spans? logs? profiles themselves?

Reply:
One task of the OTel Profiling SIG is to have some compatibility between OTel profiling and pprof (tool + protocol) - see also  https://github.com/open-telemetry/opentelemetry-proto/issues/567.

The semantic convention for OTel profiles allows to address this topic. And so the idea is to define the connection between profiles and spanes/traces within the semantic convention and remove `message Link` from the experimental OTel profiling protocol. This allows interoperability between OTel profiling and pprof.

Reply:
These compatibility layer things MUST use `otel` namespace and are generally defined elsewhere.

See: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/semantic-conventions.md#reserved-namespace